### PR TITLE
Bump Gradle 8.11 -> 8.11.1

### PR DESCRIPTION
**Pull request by wmontwe**

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -104,7 +104,7 @@ android-application = { id = "com.android.application", version.ref = "androidGr
 android-library = { id = "com.android.library", version.ref = "androidGradlePlugin" }
 android-lint = { id = "com.android.lint", version.ref = "androidGradlePlugin" }
 android-test = { id = "com.android.test", version.ref = "androidGradlePlugin" }
-compose-compiler = { id = "org.jetbrains.kotlin.plugin.compose", version.ref = "kotlinBom" }
+compose = { id = "org.jetbrains.kotlin.plugin.compose", version.ref = "kotlinBom" }
```
Comment (cketti) [inline gradle/libs.versions.toml:107]: Why does the new Gradle version require renaming this entry (but not others)?

File: settings.gradle.kts [hunk 2]
```diff
@@ -1,9 +1,9 @@
 pluginManagement {
     repositories {
         includeBuild("build-plugin")
-        gradlePluginPortal()
         google()
         mavenCentral()
+        gradlePluginPortal()
```
Comment (cketti) [inline settings.gradle.kts:6]: Why prefer plugins from Google or Maven Central over Gradle's Plugin Portal? How is this related to the Gradle version update?

File: gradle/libs.versions.toml (hunk 1 above)
Comment (wmontwe) [reply gradle/libs.versions.toml:107]: The name is wrong

File: settings.gradle.kts (hunk 2 above)
Comment (wmontwe) [reply settings.gradle.kts:6]: Because most plugin are loaded from Google and Maven repositories, this speeds up dependency resolution

File: gradle/libs.versions.toml (hunk 1 above)
Comment (cketti) [reply gradle/libs.versions.toml:107]: Maybe outdated. Although Jetbrains still seems to refer to this as the "compose compiler", e.g. here: https://www.jetbrains.com/help/kotlin-multiplatform-dev/compose-compiler.html

I don't mind the change. But this is not related to the Gradle version update at all. So please make this change a separate commit.

Comment (kewisch) [reply gradle/libs.versions.toml:107]: Do we really need to pull this out to a separate commit? Maybe you two touched on this when you discussed review rules and commits recently, but to me this is a drive by change that is acceptable without separating it out.

Comment (cketti) [reply gradle/libs.versions.toml:107]: To me it's not obvious that this is a drive-by change. It could be something that was necessitated by the Gradle update. The easy way to avoid this uncertainty and to provide some context is to separate the changes into different commits.

Comment (kewisch) [reply gradle/libs.versions.toml:107]: Technically you are correct, I'm just not sure it is worth the effort to split this out or provide context. There is some effort and overhead in splitting commits, and I don't see that it makes a difference if this is necessitated by the Gradle update or if it is just a drive-by. Is this a change you consider vital, or can we move on without?

Comment (cketti) [reply gradle/libs.versions.toml:107]: I guess now all information for this particular instance is available here in the pull request comments. So there's no immediate need to rewrite the history. 
There's a remote chance that someone investigating an issue in the future will skip this commit thinking "Bump Gradle 8.11 -> 8.11.1" is a trivial change that is unlikely to be the cause of the issue they're investigating when it was. But it's only a very remote chance.

I think reviews are important so at least one other person understands what changes make it into the app and why. That's why I ask questions when I'm not sure about the intent or effect. Conflating unrelated changes into one commit just makes things harder on everyone involved. 
Splitting things out into separate commits might make writing code take slightly longer. But I think this discussion perfectly illustrates that it can lead to a dramatic increase in the time the rest of the process takes up.
